INSTALL.md

```diff
@@ -1,5 +1,5 @@
 Building Bitcoin
 ================
 
-See doc/build-*.md for instructions on building the various
+See doc/build-*.md or [link here](https://github.com/bitcoin/bitcoin/blob/57b34599b2deb179ff1bd97ffeab91ec9f904d85/doc/README.md#building) for instructions on building the various
```

Comment:
Right, including a specific commit is id not a good idea because it essentially freezes the build instructions in time.